### PR TITLE
aligned microservice with other projects

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -19,22 +22,29 @@ var (
 )
 
 func main() {
+	err := mainWithError()
+	if err != nil {
+		panic(fmt.Sprintf("%#v\n", microerror.Mask(err)))
+	}
+}
+
+func mainWithError() error {
```
Comment (xh3b4sd) [inline main.go]: This is something I would like to do in other projects as well. It is useful to directly see the error stack in the panic message.

File: glide.yaml [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
-  - log
-  - transport/http
-- package: github.com/juju/errgo
+  version: master
+- package: github.com/giantswarm/microendpoint
+  version: master
+- package: github.com/giantswarm/microerror
+  version: master
+- package: github.com/giantswarm/micrologger
+  version: master
```
Comment (xh3b4sd) [inline glide.yaml]: We do the full thing here, endpoints, logging, error handling, all that jazz.

File: main.go (hunk 1 above)
Comment (JosephSalisbury) [inline main.go]: Why not panic in `main`?

Comment (xh3b4sd) [inline main.go]: We panic in main, a few lines above. It is just that we can properly inspect and print the error stack one time and not with every panic. And the error handling in this function can just be done as usual.

**Pull request merged**